xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -235,6 +235,9 @@ void CXBMCApp::onStart()
     intentFilter.addAction("android.net.conn.CONNECTIVITY_CHANGE");
     registerReceiver(*this, intentFilter);
     m_mediaSession.reset(new CJNIXBMCMediaSession());
+    m_activityManager.reset(new CJNIActivityManager(getSystemService(CJNIContext::ACTIVITY_SERVICE)));
```

Comment:
i might be wrong, think this is recommended
```suggestion
    m_mediaSession = std::make_unique<CJNIXBMCMediaSession>();
    m_activityManager = std::make_unique<CJNIActivityManager>(getSystemService(CJNIContext::ACTIVITY_SERVICE));
```

Reply:
jenkins already told me. ;-)

Reply:
will change this, thanks.